### ISSUE TITLE
Really revamp the onboarding

### ISSUE THREAD

**Issue by karllindmark** [ninetwozero/plastic#15]
@gustavkarlsson, how much text can one show without it being too much? Also, I don't really know what to put there. :-) #kindawroteanessay

**Comment by gustavkarlsson** [comment]
I don't see any real point keeping any screen except for the password select screen. The splash screen looks nice, but doesn't really add anything useful. The second screen describes what the app does, but shouldn't the users already know that if they downloaded the app?
Move the explanation of the encryption to the password select screen and simplify it. Also move the warning from the last screen to the password select screen. No need to explain that users will be prompted for the password either. Let them find out :)

So in short: My suggestion is to use only a single screen for the first launch and fill it with:
1. Short (!) motivation for the password
2. Simple warning about unrecoverable data on password loss
3. Password field

Possibly add a dialog with a message for testers during alpha/beta that explains how to provide feedback.

Thoughts?

**Comment by karllindmark** [comment]
@gustavkarlsson: Great feedback! Let me try some things out and get back to you... :dancers:

**Comment by karllindmark** [comment]
@gustavkarlsson: I sent you a debug APK, but it doesn't include any warning. That blue box of something is supposed to be an image of a safe, given that I manage to buy one (material style) on Fiverr.

**Comment by karllindmark** [comment]
<img src="https://cloud.githubusercontent.com/assets/979152/9832534/05f46384-597e-11e5-9c7e-e3af359d08c0.png" width="40%" height="40%">
<img src="https://cloud.githubusercontent.com/assets/979152/9832535/05f5c9cc-597e-11e5-88e9-14da5fe55387.png" width="40%" height="40%">

Current look in the debug build

**Comment by karllindmark** [comment]
Hopefully the designer will come up with something awesome for this screen. :-)

**Comment by karllindmark** [comment]
<img src="https://cloud.githubusercontent.com/assets/979152/9838963/7eaaa070-5a6b-11e5-863d-4fbb333bc004.png" width="40%" height="40%">

**Comment by gustavkarlsson** [comment]
Beautiful :)

**Comment by karllindmark** [comment]
@gustavkarlsson: Do we feel like the onboarding is revamped enough for now (bugs excluded)? If yes, I'll close this issue.

**Comment by gustavkarlsson** [comment]
Absolutely :)
